src/main/resources/application-dev.yaml

```
@@ -0,0 +1,68 @@
+spring:
+  application:
+    name: discodeit
+  boot:
+    admin:
+      client:
+        url: http://localhost:9090
+  datasource:
+    driver-class-name: org.h2.Driver
+    url: jdbc:h2:mem:testdb;MODE=PostgreSQL;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
+    username: sa
+    password:
+  h2:
+    console:
+      enabled: true
+      path: /h2-console
+  jpa:
+    hibernate:
+      ddl-auto: create
+    properties:
+      hibernate:
+        format_sql: true # SQL 줄바꿈 및 들여쓰기
+        use_sql_comments: true # 쿼리 앞에 주석 출력
+        highlight_sql: false # 컬러로 SQL 표현
+        generate_statistics: false #실행 통계 출력
+    open-in-view: false
+
+logging:
+  level:
```

`logging.level.root` 설정도 추가되면 좋을 것 같아요.
```suggestion
logging:
  level:
    root: info
```